File: C_Programming/Assignment/Assignment1/palindrome.c
#include<stdio.h>
void main()
{
    int num,r1,r2,r3,q1,res;
    printf("Enter the no:");
    scanf("%d",&num);
    r1=num%10;//121
    q1=num/10;
    r2=q1%10;
    r3=q1/10;
    
    res=r1*100+r2*10+r3;
    if(num==res)
    {
        printf("The no. is palindrome");
    }
    else
    {
        printf("The no. is not palindrome");
    }

}

//Output:Enter the no:234
// The no. is not Palindrome
// Enter the no:121
// The no. is palindrome

File: Java_Programs/Assignment1/Palindrome.java
package Assignment1;

public class Palindrome {

	public static void main(String[] args) {
		int num=121;
		int r1,r2,r3,q1,res;
//	    printf("Enter the no:");
//	    scanf("%d",&num);
	    r1=num%10;//121
	    q1=num/10;
	    r2=q1%10;
	    r3=q1/10;
	    
	    res=r1*100+r2*10+r3;
	    if(num==res)
	    {
	    	System.out.printf("The no. is palindrome");
	    }
	    else
	    {
	    	System.out.printf("The no. is not palindrome");
	    }
	}

}
